users/test_users_views.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+from django.contrib.auth.models import User
+
+
+USERNAME = "testuser"
+FIRSTNAME = "Test"
+LASTNAME = "User"
+PASSWORD = "testpass"
+EMAIL = "testuser@gmail.com"
+
+
+@pytest.fixture
+def new_user():
```

Comment:
shouldn't it be in conftest.py?

Reply:
and we also moved `test_users_views `out of `users`

Reply:
Good catch!